sign/dilithium/internal/f1600x4/f1600x4_test.go

```diff
@@ -0,0 +1,45 @@
+package f1600x4
+
+import (
+	"testing"
+)
+
+func TestF1600x4(t *testing.T) {
+	if !Available {
+		t.Skip()
+	}
+	var state State
+	a := state.Initialize()
+
+	expected := []uint64{
```

Comment:
Add a comment indicating how this was computed.

Reply:
Done.